app/models/account.rb

```diff
@@ -63,7 +63,7 @@ class Account < ApplicationRecord
   )
 
   USERNAME_RE   = /[a-z0-9_]+([a-z0-9_.-]+[a-z0-9_]+)?/i
-  MENTION_RE    = %r{(?<=^|[^/[:word:]])@((#{USERNAME_RE})(?:@[[:word:].-]+[[:word:]]+)?)}i
+  MENTION_RE    = %r{(?<!=)(?<=^|[^/[:word:]])@((#{USERNAME_RE})(?:@[[:word:].-]+[[:word:]]+)?)}i
```

Comment:
This regexp is kind of a mess. I think we may want a negative lookahead rather than a positive lookahead with a negative expression.

Maybe something like:
```suggestion
  MENTION_RE    = %r{(?<![=/[:word:]])@((#{USERNAME_RE})(?:@[[:word:].-]+[[:word:]]+)?)}i
```

Reply:
Took that suggestion and then rebased against main. I think it should be good but will watch CI.